utils/wasm-builder/test-program/build.rs

```diff
@@ -16,6 +16,11 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+use std::fs;
+
 fn main() {
     gear_wasm_builder::build();
+
+    // to be built by other tests
+    fs::write("src/rebuild_test.rs", "").unwrap();
```

Comment:
Why so complicated? Can't we just use `println!("cargo:rerun-if-...")`?

Reply:
Cargo instruction will not help because `test-program` always includes the file